experiments/run_lbann_experiment.sh

```diff
@@ -156,8 +156,8 @@ if [ -n "${IMAGENET_CLASSES}" ]; then
         catalyst|flash|quartz|surface|pascal)
             case ${IMAGENET_CLASSES} in
                 10|100|300|1000)
-                    IMAGENET_DIR=/p/lscratchf/brainusr/datasets/ILSVRC2012
-                    DATASET_TARBALLS="${IMAGENET_DIR}/resized_256x256/train.tar ${IMAGENET_DIR}/resized_256x256/val.tar ${IMAGENET_DIR}/labels.tar"
+                    IMAGENET_DIR=/p/lscratchh/brainusr/datasets/ILSVRC2012
+                    DATASET_TARBALLS="${IMAGENET_DIR}/original/train.tar ${IMAGENET_DIR}/original/val.tar ${IMAGENET_DIR}/labels.tar"
```

Comment:
These files don't exist. The lscratchf directories seem like a bit of a hot mess, so I'm not sure where, so I adamantly unvolunteer myself from resolving that... :)

Reply:
There are now tarballs in the appropriate places on lscratchh and gscratchr. I'm not sure how well the caching will work since the training set tarball is now 148 GB, compared with the 42 GB for the pre-resized training set.